rse/grant-applicants.rst

```diff
@@ -0,0 +1,144 @@
+times more efficient.  We'll also teach your group to work better.
+
+
+
+General grant considerations
+----------------------------
+
+You can find general boilerplate text to include in your proposals
+:doc:`/about/for-proposals`, but you can read below to build it in
+even more.
+
+Data Management / Open Science are big deals among some funders right
+now, and research engineers are perfect for helping with these things
+because since they focus on the how of the work, not the what.  The
+RSE service can help in the societal impact sections: your outputs
+will be more ready to be reused by society.  You could, for example,
+promise deliver more types of outputs that aren't exactly novel
+science but help society to use your results (e.g. databases,
+interactive visualisations, etc.).
+
+Make sure you mention the general Science-IT infrastructure in the
+"research environment" section, as the basic service provided by
+Aalto.  You can copy something from the boilerplate text (first link
+in this section).
+
+
+
+Academy of Finland
+------------------
+
+This applies to most general research grants, from the `general terms
+and conditions
+<https://www.aka.fi/en/research-funding/apply-for-funding/how-to-use-funding/>`__.
+Funding may be used to cover costs related to the research plan or
+action plan.  The research site must fund basic project facilities -
+which is the case at Aalto for basic RSE services.
+
+Interesting terms from the Academy: it urges research data and methods
+to be freely available.  6.2.2: "Research data and material produced
+with Academy funding in research projects and research infrastructure
+projects must be made freely available as soon as possible after the
+research results have been published."
+
+* As a service purchase:
+
+  * The research site should provide "basic project facilities", which
+    Aalto does.  Justify the extra purchase as beyond the basics.
+
+  * Please see LINK for our general costs, and how many person-months
+    you can get for a given price.  Since estimating amount of effort
+    needed is difficult, :doc:`contact us <request>` and we can
+    help you prepare an estimate.
+
+  * Maximum amount: We recommend you include no more than XXXXX as a
+    service purchase.  Please see LINK (login required) for our
+    prices, when paid via external funding.
+
+  * Justification for funding (include in proposal): "Technical
+    specialist work to ensure scientific and societal impact outputs
+    follow best practices in software development and research data
+    management practices, so that they can be of greatest possible
+    benefit to society."
+
+  * Flexibility: we will internally invoice in a way compliant with
+    Academy guidelines, and can adjust our service based on the time
+    period you need.  You don't have to decide the time period in
+    advance (only follow your submitted budget).  Our work will be
+    divided up however is useful to you.
```

Comment:
```suggestion
    divided up however if useful to you.
```

Reply:
I meant it this way but see your point.  Reworded.